### PR TITLE
Fix compiler issues

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: iron-list.html [hunk 1]
```diff
@@ -876,8 +876,8 @@
      * @return {boolean} True if the pool was increased.
      */
     _increasePoolIfNeeded: function() {
-      // Base case 1: the list has no size.
-      if (this._viewportSize === 0) {
```
Comment (justinfagnani) [inline iron-list.html]: is there test coverage for this branch?

Comment (blasten) [inline iron-list.html]: No for this branch, but for there're a few for `_increasePoolIfNeeded` https://github.com/PolymerElements/iron-list/blob/master/test/physical-count.html#L49

Comment (blasten) [inline iron-list.html]: @justinfagnani I added the test.

**Comment by frankiefu** [comment]
lgtm

**Pull request merged**